### PR TITLE
Investigate smart dashboard

### PR DESCRIPTION
**Pull request by Javonito**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -9,5 +9,7 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix-sources.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix.jar" sourcepath="/CTRE_Phoenix.sources"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CANJaguar.jar"/>
```
Comment (MrDoctorb) [inline .classpath]: Delete this.

File: .metadata/.log [hunk 2]
```diff
@@ -0,0 +1,970 @@
+!SESSION 2018-01-16 16:48:13.839 -----------------------------------------------
```
Comment (MrDoctorb) [inline .metadata/.log]: Add to gitignore

**Review by MrDoctorb — CHANGES_REQUESTED** [review]
Many of these need added to gitignore

**Comment by gowabash** [comment]
too out of date

**Pull request closed without merging**